index.html

```diff
@@ -267,20 +267,38 @@ <h2 class="title">
             placeholder="Full name"
             required
           />
+          <div class="full-name">
+            <input
+            class="first-name"
+            type="text"
+            maxlength="30"
+            name="name"
+            placeholder="First Name"
+            required
+          />
+          <input
+            class="last-name"
+            type="text"
+            maxlength="30"
+            name="last-name"
+            placeholder="Last Name"
+            required
+          />
+        </div>
           <input
             class="email"
             type="email"
             name="email"
             placeholder="Email address"
             required
           />
-          <textarea
+          <textarea class="comment"
             placeholder="Enter text here"
             name="comment"
             maxlength="500"
             required
           ></textarea>
-          <button class="form-btn" type="submit">Get in touch</button>
+          <button class="form-btn btn btn-main" type="submit">Get in touch</button>
         </form>
       </section>
       <footer class="social-footer">
```

Comment:
- [ ] [optional] Your `<footer>` tag is inside the `<main>`. It should be outside of `<main>` means it should be placed after the main. It will make sure your code is semantic and follows HTML best practices.